nautobot/core/settings.py

```diff
@@ -682,6 +682,8 @@
 # Celery results backend URL to tell workers where to publish task results
 CELERY_RESULT_BACKEND = "nautobot.core.celery.backends.NautobotDatabaseBackend"
 
+CELERY_IMPORTS = ["nautobot.core.jobs"]
```

Comment:
Add a comment as to why this is here. This is NOT intended to be a user-serviceable setting as we are using it for system jobs.